tribits/examples/TribitsSimpleExampleApp/README.md

```diff
@@ -1,10 +1,13 @@
 # TribitsSimpleExampleApp
 
-Simple example project `TribitsSimpleExampleApp` is a raw CMake project that
-pulls in libraries from a few packages from `TribitsExampleProject` using just
-`find_package(TribitsExProj REQUIRED COMPONENTS ...)`.
+This simple example project `TribitsSimpleExampleApp` is a raw CMake project
+that pulls in libraries from a few packages from
+[`TribitsExampleProject`](../TribitsExampleProject/README.md) using just
+`find_package(TribitsExProj REQUIRED COMPONENTS ...)` and also links a smaller
+program using a subset of the libraries from one of the packages from
+`TribitsExampleProject`.
```

Comment:
We can define the link again here:
[`TribitsExampleProject`](../TribitsExampleProject/README.md)

Reply:
It seems that having the first reference to `TribitsExampleProject` provide a hyperlink should be enough.  We don't want every mention of `TribitsExampleProject` on the same page to be a hyperlink.  That reduced readability.